src/index.md

```diff
@@ -1547,7 +1529,65 @@ The Skara tooling includes support for backports. [The official Skara documentat
 [To the top](#){.boxheader}
 :::
 
-# HotSpot development
+# Release Notes
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [release-note label description](#release-note)
+:::
+
+Release notes for a product (e.g. the JDK) are part of the release deliverables. They describe changes that are important for a user of the product to know about. This is usually things that may affect the user's decision to upgrade to the specific version.
+
+When writing a release note for your feature, be prepared for rather picky review comments about grammar, typos, and wording. This is for the sake of the Java community as a whole, as the language of the release note sets the tone for many blogs and news articles. For a widely used product like the JDK, the release notes are often copied (word by word, including typos) and published to highlight news in the release. This means that we need to take extra care to make sure the text in the release note is correct and has a professional language.
+
+The release note itself is written in a JBS sub-task to the issue that is used to push the change. There are a few steps to follow in order for the release note to find its way from JBS to the actual release note document.
```

Comment:
Consider: "to follow in order for the release note" -> "to follow for the release note"

Reply:
Fixed.